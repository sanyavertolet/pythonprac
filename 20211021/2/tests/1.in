:log x log(x)
log 1
:cos y cos(y)
cos 0
:e z exp(z)
e 3
:squareroot f sqrt(f)
:polynome a a**2+3*a-9
squareroot 64
polynome 5
quit 228
