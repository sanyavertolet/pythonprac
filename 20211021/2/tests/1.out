0.0
1.0
20.085536923187668
8.0
31
6:11
